test/unit/inbound/process-return-settlement.test.js

```diff
@@ -136,4 +140,45 @@ describe('process return settlement request', () => {
     }
     expect(wrapper).rejects.toThrow(Error('Database save down issue'))
   })
+
+  test('should call saveSchedule when a valid settlement is given', async () => {
+    await processReturnSettlement(settlement)
+    expect(saveSchedule).toHaveBeenCalled()
+  })
+
+  test('should call saveSchedule once when a valid settlement is given', async () => {
+    await processReturnSettlement(settlement)
+    expect(saveSchedule).toHaveBeenCalledTimes(1)
+  })
+
+  test('should call saveSchedule with "settlement" and "mocktransaction"', async () => {
+    const savedSettlement = { settlementId: 1 }
```

Comment:
Ah, I see it here -- I'm wondering if we should move this into the `beforeEach` as a 'happy path' expected return from this rather than `undefined`